docs/intro/from-ground-up.rst

```diff
@@ -160,12 +160,12 @@ The same, but using web-poet
     # === Framework-specific I/O code
     def download_sync(url):
         resp = requests.get(url)
-        return ResponseData(url=resp.url, html=resp.text)
+        return HttpResponse(url=resp.url, body=resp.content)
 
     async def download_async(session, url):
         async with session.get(url) as resp:
-            text = await resp.text()
-        return ResponseData(url=resp.url, html=text)
+            body = await resp.body
+        return HttpResponse(url=resp.url, body=body)
```

Comment:
we need to pass headers here too

Reply:
Done. https://github.com/scrapinghub/web-poet/pull/31/commits/7125ec7b8a027a37d58242847de975b14ff0839f